Resources/doc/reference/action_list.rst

```diff
@@ -525,6 +525,8 @@ You can :
 
 - `header_style`: Customize the style of header (width, color, background, align...)
 - `header_class`: Customize the class of the header
+- `cell_style`: Customize the style of inner cells (width, color, background, align...)
```

Comment:
I think so. It gives flexibility, but that's quite ugly. Not sure who uses that…